### PR TITLE
add hacktoberfest reviewer

### PR DESCRIPTION
**Pull request by MoustaphaDev**
#### What kind of changes does this PR include?
- Something else!

#### Description
Add hacktoberfest reviewer to the list

### CODE REVIEW

File: .github/hacktoberfest.md [hunk 1]
```diff
@@ -21,4 +21,5 @@ To have your review of a translation PR recognized for Hacktoberfest, open a PR
 These awesome people reviewed i18n PRs for us. Thank you! 💜
 
 - @agustinmulet reviewed #1609
-- 
+- @moustaphadev reviewed #1654
```
Comment (sarah11918) [inline .github/hacktoberfest.md:24]:
```suggestion
- @moustaphadev reviewed #1668
```

Comment (sarah11918) [reply .github/hacktoberfest.md:24]: I think this was #1668 ?  😄 I'm going to give you credit for that one, anyway!

**Comment by sarah11918** [comment]
Ooh, I think I might have selected the wrong review, since I also see you in #1654 . Please do make another PR for that one, and we'll get your name added twice, with two reviewer PRs!  🥳

File: .github/hacktoberfest.md (hunk 1 above)
Comment (MoustaphaDev) [reply .github/hacktoberfest.md]: I reviewed both, although #1668 hasn't been merged yet

**Comment by MoustaphaDev** [comment]
Sounds good 👍️

**Pull request merged**